crates/solvers/src/infra/dex/mod.rs

```diff
@@ -20,18 +22,20 @@ impl Dex {
         &self,
         order: &dex::Order,
         slippage: &dex::Slippage,
+        tokens: &auction::Tokens,
         gas_price: auction::GasPrice,
     ) -> Result<dex::Swap, Error> {
         let swap = match self {
             Dex::Balancer(balancer) => balancer.swap(order, slippage, gas_price).await?,
             Dex::OneInch(oneinch) => oneinch.swap(order, slippage, gas_price).await?,
             Dex::ZeroEx(zeroex) => zeroex.swap(order, slippage, gas_price).await?,
+            Dex::ParaSwap(paraswap) => paraswap.swap(order, slippage, tokens).await?,
```

Comment:
Interesting, ParaSwap is the only API that doesn't allow user-specified gas parameters for its route finding.

Not comment on the code per se, just something interesting I noticed (so, if as a user I want to submit a "high gas" swap, PAraSwap may pick a sub-optimal route).

Reply:
Yeah, weird...

Reply:
It appears to only allow specifying one when building the transaction and not when finding the price route (at which point, it is kind of useless anyway).